test/gui/shared/scripts/helpers/SyncHelper.py

```diff
@@ -2,7 +2,7 @@
 import re
 import sys
 import test
-import urllib
+import urllib.request
```

Comment:
We might also have to find out why the test pipeline passed despite fatal errors.